File: hex_to_ascii.py
def hex_to_ascii(hex_text):
  ascii_text = ""
  for char in hex_text:
    ascii_text += chr(int(char, 16))
  return ascii_text

if __name__ == "__main__":
  hex_text = input("Enter hexadecimal text: ")
  ascii_text = hex_to_ascii(hex_text)
  print("The ASCII representation of {} is {}".format(hex_text, 
ascii_text))

File: hex_to_ascii.swift
func hexadecimalToASCII(hexadecimal: String) -> String {
  var ascii = ""
  for char in hexadecimal {
    ascii += String(format: "%c", Int(char, radix: 16)!)
  }
  return ascii
}

let hexadecimal = "48656c6c6f2c20576f726c6421"
let ascii = hexadecimalToASCII(hexadecimal)
print("The ASCII representation of \(hexadecimal) is \(ascii)")

